server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -235,7 +280,26 @@ export class TenantManager {
     /**
      * Retrieves the secret for the given tenant
      */
-    public async getTenantKeys(tenantId: string, includeDisabledTenant = false): Promise<ITenantKeys> {
+    public async getTenantKeys(tenantId: string, includeDisabledTenant = false, disableCache=false): Promise<ITenantKeys> {
+        const lumberProperties = { [BaseTelemetryProperties.tenantId]: tenantId, includeDisabledTenant };
+        const fetchTenantKeyMetric = Lumberjack.newLumberMetric(LumberEventName.RiddlerFetchTenantKey);
+        let uncaughtException;
+        let retrievedFromCache;
```

Comment:
Nit, just move the line 291 to line 287, using `let retrevedFromCache = false` or just `let retrivedFromCache:boolean`